spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging/README.adoc

```diff
@@ -19,17 +19,30 @@ dependencies {
 }
 ----
 
-This starter registers a Web MVC interceptor that extracts the request trace ID from the `X-Cloud-Trace-Context`
-HTTP header and makes it available to logging appenders to add to log entries as metadata.
+This starter registers a Web MVC interceptor that extracts the request trace ID from the HTTP
+request and makes it available to logging appenders to add to log entries as metadata.
+
+The behavior of the trace ID extraction can be configured with the `spring.cloud.gcp.logging.traceIdExtractor`
```

Comment:
Change here to `trace-id-extractor` too.

Reply:
Ah I missed that you said it was used in the other project docs. we'll make that change in another PR.